WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainDetailsViewController.swift

```diff
@@ -0,0 +1,167 @@
+        let storyboard = UIStoryboard(name: "Domains", bundle: Bundle.main)
+        let controller = storyboard.instantiateViewController(withIdentifier: "RegisterDomainDetailsViewController") as! RegisterDomainDetailsViewController
+        return controller
+    }
+
+    private func configure() {
+        configureTableView()
+        configureNavigationBar()
+        WPStyleGuide.configureColors(for: view, andTableView: tableView)
+    }
+
+    private func configureTableView() {
+        tableView.estimatedRowHeight = Constants.estimatedRowHeight
+        WPStyleGuide.configureColors(for: view, andTableView: tableView)
+        tableView.register(
+            UINib(nibName: RegisterDomainSectionHeaderView.identifier, bundle: nil),
+            forHeaderFooterViewReuseIdentifier: RegisterDomainSectionHeaderView.identifier
+        )
+
+        ImmuTable.registerRows([EditableNameValueRow.self,
+                                CheckmarkRow.self],
+                               tableView: tableView)
+        tableHandler = ImmuTableViewHandler(takeOver: self)
+        // remove empty cells
+        tableView.tableFooterView = UIView()
+        reloadViewModel()
+    }
+
+    private func reloadViewModel() {
+        tableHandler.viewModel = tableViewModel()
+    }
+
+    private func configureNavigationBar() {
+        title = NSLocalizedString("Register domain",
+                                  comment: "Title for the Register domain screen")
+    }
+}
+
+// MARK: - Rows
+
+extension RegisterDomainDetailsViewController {
+
+    private func tableViewModel() -> ImmuTable {
+        return ImmuTable(
+            sections: [
+                privacySection(),
+                domainContactInfo()
+            ]
+        )
+    }
+
+    private func privacySection() -> ImmuTableSection {
+        var rows = [ImmuTableRow]()
+
+        rows.append(
+            CheckmarkRow(
+                title: Localized.PrivacySection.registerPrivatelyRowText,
+                checked: true,
+                action: { (row) in
+                    //TODO
+                }
+            )
+        )
+        rows.append(
+            CheckmarkRow(
+                title: Localized.PrivacySection.registerPubliclyRowText,
+                checked: false,
+                action: { (row) in
+                    //TODO
+                }
+            )
+        )
+        let section = ImmuTableSection(
+            headerText: nil,
+            rows: rows,
+            footerText: nil
+        )
+
+        return section
+    }
+
+    private func domainContactInfo() -> ImmuTableSection {
+        var rows = [ImmuTableRow]()
+
+        for fieldName in Localized.ContactInfo.fields {
+            rows.append(EditableNameValueRow(name: fieldName,
+                                             value: nil,
+                                             action: nil))
+        }
+
+        let section = ImmuTableSection(
```

Comment:
Same as above - I believe you can just do `let section = ImmuTableSection(rows: rows)` here.

Reply:
Done